main/src/main/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidator.java

```diff
@@ -72,8 +74,15 @@ public void validate(NoticeContainer noticeContainer) {
         }
         // equal shape_dist_traveled and different coordinates
         if (!(curr.shapePtLon() == prev.shapePtLon() && curr.shapePtLat() == prev.shapePtLat())) {
-          noticeContainer.addValidationNotice(
-              new EqualShapeDistanceDiffCoordinatesNotice(prev, curr));
+          double distanceBetweenShapePoints =
+              getDistanceMeters(curr.shapePtLatLon(), prev.shapePtLatLon());
+          if (distanceBetweenShapePoints >= DISTANCE_THRESHOLD) {
```

Comment:
We have to keep in mind that the distance calculated from coordinates depends on the latitude. 
So a difference of .00001 in longitude gives a distance of 1.11 m at the equator, and .8 m in Montreal.
that means that this theoretical case would give an error at the equator and a warning in Montreal.
It's an edge case, but maybe we should test for `>` instead if `>=` ?

Reply:
i'd have to change the name of the notice as well to `...bellow_or_equal_to_threshold`. Is this a change we want?

Reply:
Fine. The operative word was `maybe`